Bottom: ebb671600e8ad3cf84bb00bbad0db1b1be1a345d
Top:    ee0ed7ac0ae23595427295220e5cbc6e07876006
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-31 13:20:09 +0100

Refresh of major-re-write-to-drawing-rout

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 941d40a..44c3123 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1704,8 +1704,10 @@ ghid_draw_everything (BoxTypePtr drawn_area)
     r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback, NULL);
 
   /* Draw rat lines on top */
-  if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
+  if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0)) {
     DrawRats(drawn_area);
+    gui->set_layer (NULL, SL (FINISHED, 0), 0);
+  }
 
   Gathering = true;
